src/java/org/apache/ivy/osgi/repo/RepositoryManifestIterable.java

```diff
@@ -59,11 +59,11 @@ protected InputStream getInputStream(String f) throws IOException {
         return repo.getResource(f).openStream();
     }
 
-    protected List<String> listBundleFiles(String dir) throws IOException {
+    protected List<String> listBundleFiles(String dir) {
         return asList(ResolverHelper.listAll(repo, dir));
     }
 
-    protected List<String> listDirs(String dir) throws IOException {
+    protected List<String> listDirs(String dir) {
```

Comment:
Similar comment as here https://github.com/apache/ant-ivy/pull/52/files#r127395633 applies to this line